src/main/java/edu/ie3/datamodel/io/processor/result/ResultEntityProcessor.java

```diff
@@ -68,7 +70,7 @@ yield quantityValToOptionalString(
       case "q":
         yield quantityValToOptionalString(
             quantity.asType(Power.class).to(StandardUnits.REACTIVE_POWER_RESULT));
-      case "p", "pMax", "pOwn", "pThermal":
+      case "p", "pMax", "pOwn", "pThermal", "preference", "pmin", "pmax":
```

Comment:
Yes, they should, and they should've also been part of `Processor#specificQuantityFieldNames`. I fixed all that and adapted `ResultEntityProcessorTest` to fail otherwise